### PR TITLE
Undeprecate Asset Pricing III

### PR DESCRIPTION
**Pull request by arnavs**
This is based on #43. Updates the code and moves in the code that we were including from `_static/code/harrison_kreps`, since we weren't using it anywhere else.

### CODE REVIEW

File: rst_files/harrison_kreps.rst [hunk 1]
```diff
@@ -435,8 +455,40 @@ Investors of type :math:`a` want to sell the asset in state :math:`1` while inve
 
 Here's code to solve for :math:`\bar p`, :math:`\hat p_a` and :math:`\hat p_b` using the iterative method described above
 
-.. literalinclude:: /_static/code/harrison_kreps/hk_price_optimisticbeliefs.jl
+.. code-block:: julia 
+
+    """
+    Function to Solve Optimistic Beliefs
+    """
```
Comment (Nosferican) [inline rst_files/harrison_kreps.rst]: Delete docstring

File: rst_files/harrison_kreps.rst [hunk 2]
```diff
@@ -309,8 +309,28 @@ The first two rows of of the table report :math:`p_a(s)` and :math:`p_b(s)`
...
 
+    Authors: Shunsuke Hori
+
+    =#
+    using LinearAlgebra 
+
+    """
+    Function to Solve Single Beliefs
+    """
+    function price_single_beliefs(transition::Matrix, dividend_payoff::Vector;
```
[the remark this answers is not in the document]
Comment (Nosferican) [reply rst_files/harrison_kreps.rst]: And throughout. Always good to use the abstract unless one is dispatching.

File: rst_files/harrison_kreps.rst (hunk 1 above)
Comment (arnavs) [reply rst_files/harrison_kreps.rst]: OK done. I'll do these when I catch them, but in general I think we should reserve these PRs for functional improvements.

File: rst_files/lucas_model.rst [hunk 3]
```diff
@@ -414,12 +414,109 @@ Let's try this when :math:`\ln y_{t+1} = \alpha \ln y_t + \sigma \epsilon_{t+1}`
...
+    @authors : Spencer Lyon <spencer.lyon@nyu.edu>, John Stachurski
+
+    =#
+
+    using QuantEcon, Distributions
+
+    """
+    The Lucas asset pricing model --- parameters and grid data
+    """
+    struct LucasTree{TF<:AbstractFloat}
```
Comment (Nosferican) [inline rst_files/lucas_model.rst]: This def needs be re-written afterwards.

Comment (arnavs) [reply rst_files/lucas_model.rst]: For sure. Lots of boilerplate that we can eliminate.

**Pull request merged**